pkg/kubelet/cm/dra/manager.go

```diff
@@ -304,7 +290,7 @@ func (m *ManagerImpl) GetResources(pod *v1.Pod, container *v1.Container) (*Conta
 // already been successfully unprepared.
 func (m *ManagerImpl) UnprepareResources(pod *v1.Pod) error {
 	batches := make(map[string][]*drapb.Claim)
-	claimInfos := make(map[types.UID]*ClaimInfo)
+	claimInfoNames := make(map[types.UID]string)
```

Comment:
```suggestion
	claimNames := make(map[types.UID]string)
```

Reply:
Out of curiosity, though -- why not just track the claimInfo since we are getting it out of the cache anyway? That way we could remain symmetrical with `PrepareResources()`?

Reply:
The idea was to use it only where it's actually needed. It's still not obvious that claimInfo is a deep copy, which can cause confusion. It's the opposite of what we had - claimInfo was a reference to a cached object, but it was not obvious, and people were updating it without thinking about cache consistency.